source/ScrollVar.h

```diff
@@ -0,0 +1,72 @@
+/* ScrollVar.h
```

Comment:
Either remove this class or use it in all the places that need scrolling.

Reply:
The reason why I added this class was because I kept reusing these chunks of code over and over again. I intend to submit future PRs to handle scrolling in other places as well, but I consider that out of scope for this PR.